NOTICE.txt

```diff
@@ -190,16 +190,6 @@ Apache License 2.0
 
 
 --------------------------------------------------------------------
-Dependency: github.com/elastic/beats
-Version: master
-Revision: 57aaa90929a49ba9b11cdc800c49930df3e24161
-License type (autodetected): Apache License 2.0
-./vendor/github.com/elastic/beats/LICENSE.txt:
---------------------------------------------------------------------
-Apache License 2.0
-
-
---------------------------------------------------------------------
```

Comment:
why is this removed? shouldn't it be re-added after updating beats?

Reply:
Not 100% sure what happened here before but it seems it was a duplicate, as it's also a few lines up (187).